tools/project_manager/show_project_wordcheck_stats.php

```diff
@@ -26,9 +26,7 @@
 // load bad words for the site for this project
 $site_bad_words = load_site_bad_words_given_project($projectid);
 
-// get the latest possible round
-$last_possible_round = get_Round_for_round_number(MAX_NUM_PAGE_EDITING_ROUNDS);
-$pages_res = page_info_query($projectid, $last_possible_round->id, 'LE');
+$pages_res = page_info_query($projectid, Rounds::get_last()->id, 'LE');
```

Comment:
I'm getting 

```Notice: Undefined variable: last_possible_round in /home/cpeel/public_html/c.branch/remove-max-rounds-const/tools/project_manager/show_project_wordcheck_stats.php on line 62

Notice: Trying to get property 'id' of non-object in /home/cpeel/public_html/c.branch/remove-max-rounds-const/tools/project_manager/show_project_wordcheck_stats.php on line 62
'' is not a valid round-id 
```
(but don't know how to quote that line specifically -- guess I need some instruction!)

Reply:
Whoops, I missed the second use of that variable on the page. Fixed!